pkg/apis/pipeline/v1alpha1/workspace_validation_test.go

```diff
@@ -77,6 +85,30 @@ func TestWorkspaceBindingValidateInvalid(t *testing.T) {
 				ClaimName: "pool-party",
 			},
 		},
+	}, {
+		name: "Provided both emptydir and configmap",
```

Comment:
Added in response to https://github.com/tektoncd/pipeline/pull/1800#discussion_r366455223

We could write a thing that tests every permutation but I'm not super compelled to do so with this PR.